Makes some changes to the default LabWC (Xfce Wayland) configuration,
so as to be better suited for MassOS. Also brings back some default
keyboard shortcuts from Xfwm4 which are missing in LabWC by default.

--- a/labwc/labwc-rc.xml
+++ b/labwc/labwc-rc.xml
@@ -2,17 +2,22 @@
 <!--
   This is a very simple config file with many options missing. For a
   complete set of options with comments, see Labwc docs/rc.xml.all
+
+  Some modifications to upstream defaults have been made for MassOS.
+
+  After modifying this file, you must run 'labwc -r', or re-login.
 -->
 <labwc_config>
 
+  <!-- Change to 'automatic' or 'cascade' if you prefer. -->
   <placement>
-    <policy>automatic</policy>
+    <policy>center</policy>
   </placement>
 
   <theme>
-    <name>Adwaita</name>
+    <name>Arc-Dark</name>
     <cornerRadius>8</cornerRadius>
-    <font name="sans" size="10"/>
+    <font name="sans" size="11"/>
   </theme>
 
   <desktops number="4" />
@@ -23,6 +28,10 @@
     <keybind key="W-Return">
       <action name="Execute" command="xfce4-terminal"/>
     </keybind>
+    <!-- Also provide Ctrl+Alt+T, which is a default in Xfwm4. -->
+    <keybind key="C-A-T">
+      <action name="Execute" command="xfce4-terminal"/>
+    </keybind>
     <!--
       Remove a previously defined keybind
       A shorter alternative is <keybind key="W-F4" />
@@ -31,19 +40,21 @@
       <action name="None"/>
     </keybind>
 
-    <numlock>on</numlock>
+    <!-- Upstream defaults numlock to on. We default to off. -->
+    <numlock>off</numlock>
 
     <!--
        Suggested Xfce key bindings.
     -->
-    <keybind key="XF86_AudioLowerVolume">
-      <action name="Execute" command="amixer -D pulse sset Master 5%-" />
+    <!-- amixer is not usable for PipeWire. Use wpctl instead. -->
+    <keybind key="XF86AudioLowerVolume">
+      <action name="Execute" command="wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%-" />
     </keybind>
-    <keybind key="XF86_AudioRaiseVolume">
-      <action name="Execute" command="amixer -D pulse sset Master 5%+" />
+    <keybind key="XF86AudioRaiseVolume">
+      <action name="Execute" command="wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%+" />
     </keybind>
-    <keybind key="XF86_AudioMute">
-      <action name="Execute" command="amixer -D pulse sset Master toggle" />
+    <keybind key="XF86AudioMute">
+      <action name="Execute" command="wpctl set-mute @DEFAULT_AUDIO_SINK@ toggle" />
     </keybind>
     <keybind key="W-q">
       <action name="Close" />
@@ -66,6 +77,16 @@
     <keybind key="W-4">
       <action name="GoToDesktop" to="4" />
     </keybind>
+    <!-- Additional shortcuts which are default in Xfwm4. -->
+    <keybind key="A-F2">
+      <action name="Execute" command="xfce4-appfinder --disable-server" />
+    </keybind>
+    <keybind key="W-r">
+      <action name="Execute" command="xfce4-appfinder --disable-server -c" />
+    </keybind>
+    <keybind key="Print">
+      <action name="Execute" command="xfce4-screenshooter" />
+    </keybind>
   </keyboard>
 
   <snapping>
